code/controllers/subsystem/processing/quirks.dm

```diff
@@ -29,6 +29,7 @@ GLOBAL_LIST_INIT_TYPED(quirk_blacklist, /list/datum/quirk, list(
 	list(/datum/quirk/no_guns, /datum/quirk/bighands, /datum/quirk/poor_aim),
 	list(/datum/quirk/no_guns, /datum/quirk/nonviolent),
 	list(/datum/quirk/spacer_born, /datum/quirk/oversized),
+	list(/datum/quirk/item_quirk/gun_nut, /datum/quirk/no_guns, /datum/quirk/nonviolent, /datum/quirk/equipping/nerve_staple, /datum/quirk/insanity, /datum/quirk/unstable)
```

Comment:
Quirk blacklists like this prevent all quirks from being picked with each other. This would make someone be unable to take ANY combination of the 6 quirks together, which means you can't be someone who is nonviolent and has RDS now. They need to be split into separate lines. Yes, it's jank.

Reply:
Didn't know that; changing rn.